ci/docker/bosh-google-cpi-boshrelease/Dockerfile

```diff
@@ -50,6 +50,9 @@ RUN ruby-install ruby 2.1.2 && \
 # Install Bundler and BOSH CLI
 RUN /bin/bash -l -c "gem install bundler bosh_cli --no-ri --no-rdoc"
 
+# Install AWS CLI (used to upload stemcell)
+RUN pip install awscli
```

Comment:
Yes, we build and push the new Docker image after merge; we have not done that yet. Let us know when you've merged & we'll be happy to do it.